src/main/scala/io/iohk/ethereum/vm/BlockchainConfigForEvm.scala

```diff
@@ -68,7 +71,9 @@ object BlockchainConfigForEvm {
     val BeforeByzantium, Byzantium, Constantinople, Petersburg, Istanbul, Berlin = Value
   }
 
-  def isEip2929Enabled(etcFork: EtcFork): Boolean = etcFork >= EtcForks.Magneto
+  def isEip2929Enabled(etcFork: EtcFork, ethFork: BlockchainConfigForEvm.EthForks.Value): Boolean =
```

Comment:
It could be nicer to use EthForks.EthFork instead of EthForks.Value (or better with the proper import just use EthFork) to mimick the EtcFork type declaration

Reply:
I agree, but for some reason the compiler is unhappy with what you proposed

Reply:
Strange, you're right. It can handle explicit EthForks.EthFork but not import + standalone EthFork

Reply:
And for whatever reason, even thought I can't see a difference, removing the manual EthFork import, and letting idea fixes itself the missing import seems to do the tricks.